docs/api.rst

```diff
@@ -0,0 +1,104 @@
+API Reference
+=============
+
+Core Function
+-------------
+
+parse
```

Comment:
How do u feel about,
- Adding a page to include the benchmark report?
- Define metric(similarity) for evaluation?

Reply:
Added